### PR TITLE
Update the layout of the About the Project section on the landing page (resolves #4)

### LINKED ISSUE #4: Update/fix the layout of the About the Project section on the landing page
Priority 2
Update the design of the About the Project section on the landing page according to the mockups. Option 1 if possible, otherwise Option 2. Dana to provide high res image of the map, hopefully @amb26 can provide one.
Option 1:
[
about the project option 1
](url)
Option 2:
about the project option 2

Figma link (see page - Landing Page Sept 2024): https://www.figma.com/design/yXKyjRyHbnJPB6jrbfO0dq/Xetthecum?node-id=4059-2382&t=BlPJcpGn9NAMxoJY-1

### PR DESCRIPTION
**Pull request by jobara**
Resolves #4 

- Updates the layout for the About the Project section using Option 1
  - Uses a large background image of the map
  - Places the content of the About the Project section over the map with a slightly transparent background

### CODE REVIEW

**Comment by dayotte** [comment]
😃 Thank you @jobara !

**Comment by amb26** [comment]
I find the text in the new section rather hard to read - could we do something about the contrast?

image

I've merged your #20 already so do merge this up, thanks!

**Comment by dayotte** [comment]
@jobara @amb26 the text is missing the background box
image

File: css/styles.css [hunk 1]
```diff
@@ -114,6 +114,25 @@ p {
...
+  padding-inline: 3em;
+  margin-block: clamp(1.5em, calc(100vw / 20), 5em);
+}
+
+.floating-text .btn {
+  background-color: var(--bs-body-color);
+  color: rgba(245, 251, 219, 1);
+  font-size: 1.2em;
+  margin-inline: 0;
+}
```
Comment (jobara) [inline css/styles.css:134]: @amb26 related to your comment about the contrast, it seems these styles or at least some of them aren't being applied for you. Not sure why though.

**Pull request merged**